src/main/java/com/echobox/api/linkedin/util/ValidationUtils.java

```diff
@@ -88,4 +91,34 @@ public static void validateResponse(WebRequestor.Response response) {
       throw new LinkedInNetworkException("LinkedIn request failed", response.getStatusCode());
     }
   }
+  
+  /**
+   * Validate that the response contains the required header.
+   *
+   * @param headers map of headers from the response
+   * @param header the header to check exists
+   */
+  public static void validateRequiredResponseHeader(Map<String, String> headers, String header) {
+    if (headers == null || headers.isEmpty()) {
+      throw new LinkedInResponseException("No headers were found on the response.");
+    }
+  
+    String headerResponse = headers.get(header);
+    if (headerResponse == null) {
+      throw new LinkedInResponseException(String.format("The header [%s] is missing from the "
+          + "response.", header));
+    }
+  }
+  
+  /**
+   * Validate video files
+   *
+   * @param fileSizeBytes size of the video file in bytes
+   */
+  public static void validateVideoFile(long fileSizeBytes) {
+    if (fileSizeBytes > Integer.MAX_VALUE) {
```

Comment:
May I know where you found this file size limit?

Reply:
The method used to chunk the video file accepts `int` as parameters. If the file is larger than what an int is able to hold, the chunks that overlap that limit will throw an exception. At the moment, that gives us a limit of being able to support a file of ~2GB

Reply:
I found below from LinkedIn documentation, but they conflict each other
`200MB`: https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/videos-api?view=li-lms-2023-01&tabs=http#video-file-size-specifications
`5GB`: https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/videos-api?view=li-lms-2023-01&tabs=http#schema
(in schema - `initializeUploadRequest.fileSizeBytes`)

Reply:
I was only aware of the 5GB limit. I will test to confirm.

Reply:
Based on a call to their API, the limit is 5GB. I received the following error when specifying a 6GB file size:
```
File size for video must be between 1 and 5368709120 bytes
```

Reply:
Should below condition be revised?
`    if (fileSizeBytes > Integer.MAX_VALUE) {`

Reply:
I don't think it's necessary at this point. 
Our video file size limit would be ~2GB, which I think should be sufficient. We can iterate at a later stage if we need to.

Reply:
Thanks for your elaboration. As discussed, Please put the limit (Integer.MAX_VALUE) as a constant, and state why we put 2GB here, instead of 5GB